src/ui/views/ViewMobileOnboarding.qml

```diff
@@ -260,29 +260,31 @@ VPNFlickable {
             Layout.minimumHeight: VPNTheme.theme.windowMargin
         }
 
-        ColumnLayout {
+        Column {
```

Comment:
Yeaa, I had a feeling making this a layout would break something.

![giphy](https://user-images.githubusercontent.com/15353801/165605650-7294f6a2-44ff-47be-b9e3-36346ac7b6a2.gif)

I was having trouble getting the spacing to match the spec with a positioner, but didn't notice the binding loop. I think this looks good nonetheless!

Reply:
haha nw it's too easy to do! I'd love to find a more resilient approach to responsive QML layouts.